### PR TITLE
Update constraints example. Fixes #23

### LINKED ISSUE #23: Update constraints example
Update constraints example to use API instead of CLI and consolidate maxmutation into constraints example.

### PR DESCRIPTION
**Pull request by milleruntime**

### CODE REVIEW

File: docs/constraints.md [hunk 1]
```diff
@@ -20,35 +20,27 @@ This tutorial uses the following Java classes, which can be found in org.apache.
...
+    $ ./bin/runex constraints.NumericValueConstraint
+
+The MaxMutationSize constraint will force the table to reject any mutation that is larger than 1/256th of the
+working memory of the tablet server.  The following example attempts to ingest a single row with a million columns,
+which exceeds the memory limit. Depending on the amount of Java heap your tserver(s) are given, you may have to
+increase the number of columns provided to see the failure.
+
+    $ ./bin/runex constraints.MaxMutationSize
+
+[AlphaNumKeyConstraint.java]: ../src/main/java/org/apache/accumulo/examples/constraints/AlphaNumKeyConstraint.java
```
Comment (keith-turner) [inline docs/constraints.md]: this is unused

Comment (keith-turner) [inline docs/constraints.md:41]: This is unused

File: docs/constraints.md [hunk 2]
```diff
@@ -20,35 +20,27 @@ This tutorial uses the following Java classes, which can be found in org.apache.
 
  * AlphaNumKeyConstraint.java - a constraint that requires alphanumeric keys
  * NumericValueConstraint.java - a constraint that requires numeric string values
+ * MaxMutationSize.java - a constraint that limits the size of mutations accepted into a table
+
+Remember to copy the accumulo-examples-\*.jar to Accumulo's 'lib/ext' directory.
+
+The two examples below create table constraints for a key and value.  The AlphaNumKeyConstraint will prevent
```
Comment (keith-turner) [inline docs/constraints.md]: Could make this less wordy... below is a quick attempt at this

```
AlphaNumKeyConstraint prevents insertion of keys with characters not between aA and zZ or 0 to 9.  NumericValueConstraint￼ prevents insertion of values from with characters not between 0 and 9. The examples create mutations that violate the constraint, throwing an exception.
```

File: docs/constraints.md (hunk 1 above)
Comment (milleruntime) [reply docs/constraints.md:41]: Forgot to actually use the links... whoops

File: docs/constraints.md [hunk 3]
```diff
@@ -18,16 +18,15 @@ limitations under the License.
...
+ * [MaxMutationSize.java] - a constraint that limits the size of mutations accepted into a table
 
 Remember to copy the accumulo-examples-\*.jar to Accumulo's 'lib/ext' directory.
 
-The two examples below create table constraints for a key and value.  The AlphaNumKeyConstraint will prevent
-any keys from being created with characters that are not between aA and zZ or 0 to 9.  The NumericValueConstraint will
-prevent any values from being created with characters that are not between 0 and 9. The examples will attempt to create
-key value pairs for the table that violate the constraint, throwing an exception.
+AlphaNumKeyConstraint prevents insertion of keys with characters not between aA and zZ or 0 to 9.  
+NumericValueConstraint￼prevents insertion of values with characters not between 0 and 9. The examples create mutations
```
Comment (keith-turner) [inline docs/constraints.md]: missing a space `NumericValueConstraint￼prevents`

**Review by keith-turner — APPROVED** [review]

**Pull request merged**